frontend/online-shop/src/App.tsx

```diff
@@ -1,26 +1,30 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import { TableCards } from "./components/productsList/TableCards";
+import { ProductDetails } from "./components/productDetails/ProductDetails";
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Redirect,
+} from "react-router-dom";
+import { ThemeProvider } from "@material-ui/styles";
+import { standardTheme } from "./themes/StandardTheme";
```

Comment:
As long as you have provided a custom theme it's confusing to leave the name `standardTheme`. `customTheme` would fit much better.